src/main/java/org/jitsi/jicofo/xmpp/FocusComponent.java

```diff
@@ -258,7 +253,7 @@ public IQ handleIQSetImpl(IQ iq)
                 org.jivesoftware.smack.packet.IQ response
                     = handleConferenceIq((ConferenceIq) smackIq);
 
-                return response != null ? IQUtils.convert(response) : null;
+                return IQUtils.convert(response);
```

Comment:
how about the `null` check? IQUtils.convert doesn't seem to be safe in that regard.

Reply:
`handleConferenceIq` never returns null (it returns an error IQ if something goes wrong). That was detected by my IDE, and I marked it with `@NotNull`